code/modules/projectiles/projectile.dm

```diff
@@ -506,6 +506,8 @@
 			return FALSE
 		if(HAS_TRAIT(L, TRAIT_IMMOBILIZED) && HAS_TRAIT(L, TRAIT_FLOORED) && HAS_TRAIT(L, TRAIT_HANDS_BLOCKED))
 			return FALSE
+		if(L.body_position == LYING_DOWN && !L.has_status_effect(STATUS_EFFECT_CRAWLING))
+			return FALSE
 		if(!hit_prone_targets)
```

Comment:
This variable should be recalled to `hit_crawling_targets` and line 514-515 should also be removed for being redundant and useless.

Reply:
good point